frontend/src/components/HacknightsParticipants.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <v-container fluid text-xs-center>
+    <v-row align="center" justify="center">
+      <v-col class="d-flex" cols="12" sm="6">
+        <v-combobox
+          v-model="selectedParticipants"
+          outlined
+          :items="
+            getParticipants.filter(
+              participant =>
+                !getHacknight.participants.filter(y => y.id === participant.id)
+                  .length
+            )
+          "
+          item-text="email"
+          item-value="email"
+          :search-input.sync="search"
+          hide-selected
+          label="Add participants"
```

Comment:
Inside the app we use Polish :P please change to "dodaj użytkowników"

The same goes for hacknight component line 31 where instead of "New" should stand "Nowy hacknight" - I didn't catch it then before merging.

Reply:
Should we ? It will be inconsistent at least to the moment when we introduce some tool for internationalization to backend eg. Flask-Babel. But even now with english only we struggle to provide consistent messages to the frontent client. We could also intercept those messages and translate them somewhere within frontend, but until now no one came with such idea.

Reply:
I don't think I understand what you mean. Are those labels somehow related to messages? In the comment I assumed that they only display the string in the button and I figured that since (apart from github) we speak mostly in Polish it will be more natural. Also the potential inconsistency between languages in content and messages personally doesn't bother me at all, but maybe it's just me :)